index.bs

```diff
@@ -730,6 +732,10 @@ Any time the user agent obtains a new [=sensor reading=] for a [=platform sensor
 platform, it invokes [=update latest reading=] with the [=platform sensor=] and the [=sensor reading=]
 as arguments.
 
+Any time a new [=sensor reading=] for a [=platform sensor=] is obtained, user agent must check if
```

Comment:
Any time a new [=sensor reading=] for a [=platform sensor=] is obtained and if the user agent [=can expose sensor readings=] ..